cmd/app.go

```
@@ -92,33 +177,34 @@ for the specified resource type.`,
 	}
 
 	fingerprintCmd.Flags().StringSlice("targets", []string{}, "URL target to perform fingerprint against")
-	fingerprintCmd.Flags().String("fingerprintfile", "configs/app/fingerprints.json", "Path to the fingerprint file to use for fingerprinting")
-	fingerprintCmd.Flags().String("resourcetype", "", "Defined resource type to fingerprint")
+	fingerprintCmd.Flags().String("fingerprint-file", "configs/app/fingerprints.json", "Path to the fingerprint file to use for fingerprinting")
+	fingerprintCmd.Flags().String("resource-type", "", "Defined resource type to fingerprint")
 	fingerprintCmd.Flags().StringSlice("modules", []string{}, "Defined resource type modules to run")
+	fingerprintCmd.Flags().Bool("successful-only", false, "Only show successful attempts")
+	fingerprintCmd.Flags().Bool("insecure", false, "Allow insecure SSL connections and transfers")
 	fingerprintCmd.Flags().Int("timeout", 30, "Timeout per request (seconds)")
-	fingerprintCmd.Flags().Bool("successfulonly", false, "Only show successful attempts")
 
 	_ = fingerprintCmd.MarkFlagRequired("targets")
-	_ = fingerprintCmd.MarkFlagRequired("resoursetype")
+	_ = fingerprintCmd.MarkFlagRequired("resource-type")
```

lets add panic handlers for missing flags right here:
```
if err := fuzzCmd.MarkFlagRequired("targets"); err != nil {
		panic(err)
	}
```

---

func (c *Command) MarkFlagRequired(name string) error {
	return MarkFlagRequired(c.Flags(), name)
}

Doesnt throw a panic, its actually an error that gets caught in the signal and returned